partcad/src/partcad/ai_google.py

```
@@ -123,30 +155,39 @@ def handle_content(content: AiContentFile):
+            max_output_tokens=tokens,
+            temperature=temperature,
+            top_p=top_p,
+            top_k=top_k,
         )
+
         candidates = []
         options_left = options_num
         while options_left > 0:
+            response = None
             retry = True
             while retry == True:
```

Ruff (0.15.21)

[error] Avoid equality comparisons to `True`; use `retry:` for truth checks

Replace with `retry`

(E712)